### DISCUSSION TITLE
Check Birdnet-Pi from a distant computer

### DISCUSSION THREAD

**Discussion by Fish31520** [mcguirepr89/BirdNET-Pi#570] [Q&A] [answered]
Hi,

One more question for you folks :

My girlfriend doesn't live in the same county as me. Is there a way for her to check my Birdnet recordings from her computer (I'm absolutely not familiar with ssh things so my question may sound super dumb ! 😅) ?

Thanks !

**Comment by ehpersonal38** [comment, marked as answer]
Hey there!

This is absolutely possible. Patrick has created a guide that I hope you will find useful: https://github.com/mcguirepr89/BirdNET-Pi/wiki/Sharing-Your-BirdNET-Pi

Let me know if you have any questions or need any help!

**Reply by Fish31520** [reply]
Nice, thanks a lot !

It's too late for tonight (1 am 😅) but I gonna take a look at that tomorrow ! 😉

**Reply by Fish31520** [reply]
Hmmmm, it's harder than I thought.

Well, my girlfriend is going to live with me in few months so... I guess we can wait a bit ! 😆

Thanks a lot by the way !
